.gitignore

```diff
@@ -16,20 +29,25 @@
 # Settings
 /setup.cfg
 
+# OS
+.DS_Store
+
 # Star catalogues
 tetra3/BSC5*
 tetra3/hip_main.dat
 tetra3/tyc_main.dat
 
 # Output dirs
 examples/data/medium_fov/output
+!examples/data/medium_fov/output/.keep
```

Comment:
the output folder is required to exist for `examples/test_tetra3.py` to work so the convention to force git to make a directory is a hidden `.keep` file